### PR TITLE
Add Vulkan and widevine blobs for kenzo

### PR DESCRIPTION
**Pull request by poad42**

### CODE REVIEW

**Comment by dadi11** [comment]
why are you kanging @minz1 commits?

File: kenzo/kenzo-vendor.mk [hunk 1]
```diff
@@ -133,6 +133,9 @@ PRODUCT_COPY_FILES += \
     vendor/xiaomi/kenzo/proprietary/lib64/hw/consumerir.msm8952.so:system/lib64/hw/consumerir.msm8952.so \
     vendor/xiaomi/kenzo/proprietary/lib64/hw/fingerprint.msm8952.so:system/lib64/hw/fingerprint.msm8952.so \
     vendor/xiaomi/kenzo/proprietary/lib64/lib_fpc_tac_shared.so:system/lib64/lib_fpc_tac_shared.so \
+    vendor/xiaomi/kenzo/proprietary/lib/egl/egl.cfg:system/lib/egl/egl.cfg \
```
Comment (luca020400) [inline kenzo/kenzo-vendor.mk]: egl.cfg. What android version is that, gingerbread?

**Comment by poad42** [comment]
Fixed !

**Comment by nicknitewolf** [comment]
Code Review -2

**Comment by bgcngm** [comment]
Closing based on: https://github.com/TheMuppets/proprietary_vendor_xiaomi/pull/30.

**Pull request closed without merging**